src/driver/uart.rs

```diff
@@ -0,0 +1,50 @@
+//! UART Driver
+//!
+//! This module implements a driver for the UART driver. This driver allows Miralis to communicate with the UART.
+
+use core::fmt::Write;
+use core::{fmt, ptr};
+
+pub struct UartDriver {
+    serial_port_base_addr: usize,
+    size_per_register: usize,
+}
+
+impl UartDriver {
+    pub const fn new(serial_port_base_addr: usize, size_per_register: usize) -> Self {
+        UartDriver {
+            serial_port_base_addr,
+            size_per_register,
+        }
+    }
+
+    pub const fn get_register(&mut self, offset: usize) -> usize {
+        // Registers are 32 bits wide on the board
```

Comment:
What does this comment mean? Aren't the register size read of `self.size_per_register`?

Reply:
Ho I see, that code comes from the VisionFive2, we can simply remove the comment then :)